src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0,   minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <div class="card" data-qa="card">
+        <div class="card__img-wrap">
+          <img
+          class="imac-photo"
+          src="images/imac.jpeg"
+          alt="imac photo"
+          >
+        </div>
+
+        <h3 class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">
+          Код товара: 195434
+        </p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <span class="card__amount-reviews">Отзывов: 5</span>
+        </div>
+
+        <div class="card__price">
+          <span>Цена:</span>
+          <span class="card__price-amount">69 999 грн</span>
+        </div>
+
+        <div class="card__buying">
```

Comment:
not necessary `<div class="card__buying">`, so you can remove it.

Reply:
card__buying is the parent element. it for margins